libraries/testing/include/eosio/testing/tester.hpp

```diff
@@ -343,7 +343,7 @@ namespace eosio { namespace testing {
                   const auto& accnt = control->db().get<account_object, by_name>( name );
                   abi_def abi;
                   if( abi_serializer::to_abi( accnt.abi, abi )) {
```

Comment:
```suggestion
                  if( abi_def abi; abi_serializer::to_abi( accnt.abi, abi )) {
```

Reply:
Same change can be done to other locations as well. It helps enfoce the fact that the moved from variable is not reused.